lib/label.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'item'
+require 'date'
+
+class Label < Item
+  attr_accessor :title, :color
+  attr_reader :id, :items, :publish_date
+
+  def initialize(title, color, items, publish_date)
+    # Calling Constructor of Parent Class (Item)
+    super('#label', '#author', '#genre', publish_date)
+    @id = rand(1...100)
```

Comment:
Hey @MasumaJaffery, great job so far!
## Highlights
- Met all technical requirement ✅ 
## Issue ♻️ 
- I think that since the `id` is managed by the `Item` class, you don't need to add it again here in the class `Label`.